Resources/Prototypes/Entities/Objects/base_item.yml

```diff
@@ -36,3 +36,14 @@
     noRot: false
   - type: Pullable
   - type: ExplosionLaunched
+
+- type: entity
+  parent: BaseItem
+  id: BaseItemSmall
```

Comment:
Might want to make this BaseItemSmallTrashable, since it might be mistakenly put on other items like pens.

Reply:
Aight, I was actually going to put it on pens but they already had a size component so didn't. Will correct